sale_order_partial_invoice/sale.py

```diff
@@ -47,12 +47,14 @@
 
 """
 import logging
-_logger = logging.getLogger(__name__)
```

Comment:
Don't we want to keep a blank line here to separate openerp import from lib imports?